app/renderer/components/bookmarksToolbar.js

```diff
@@ -47,6 +48,21 @@ class BookmarkToolbarButton extends ImmutableComponent {
     }
   }
 
+  onMouseOver (e) {
+    // Behavior when a bookmarks toolbar folder has its list expanded
+    if (this.props.selectedFolderId) {
```

Comment:
I think that ideally this should trigger something like an `onMouseOverBookmarkFolder` action with whatever params are needed to allow the windowstore (browser process eventually) to update the state rather than having the logic here to open context menus and update state. The are two reasons for this:
1. It's more inline with how flux is intended to work. We should have declarative actions instead of imperative
2. We want to take node out of the main window renderers so we want to move electron api calls out of the react components (showBookmarkFolderMenu indirectly calls `electron.remote` methods) in preparation for moving them to the browser process